File: mifosng-android/src/main/java/com/mifos/api/BaseUrl.java
/*
 * This project is licensed under the open source MPL V2.
 * See https://github.com/openMF/android-client/blob/master/LICENSE.md
 */

package com.mifos.api;

/**
 * @author fomenkoo
 */
public class BaseUrl {

    public static final String PROTOCOL_HTTPS = "https://";
    public static final String API_ENDPOINT = "65.2.12.252";
    public static final String API_PATH = "/fineract-provider/api/v1/";
    public static final String PORT = "8443";
    // "/" in the last of the base url always

    public String getName() {
        return "mifos";
    }
}

File: core/network/src/main/java/com/mifos/core/network/BaseUrl.kt
/*
 * This project is licensed under the open source MPL V2.
 * See https://github.com/openMF/android-client/blob/master/LICENSE.md
 */
package com.mifos.core.network

import com.mifos.core.common.BuildConfig
import com.mifos.core.common.utils.asServerConfig

/**
 * @author fomenkoo
 */
class BaseUrl {

    // "/" in the last of the base url always
    companion object {
        private val configs = BuildConfig.DEMO_SERVER_CONFIG.asServerConfig()

         val PROTOCOL_HTTPS = configs.protocol
         val API_ENDPOINT = configs.endPoint
         val API_PATH = configs.apiPath
         val PORT = configs.port
    }
}